.github/workflows/execution-performance.yaml

```diff
@@ -8,15 +8,15 @@ on:
 
 jobs:
   execution-performance:
-    uses: aptos-labs/aptos-core/.github/workflows/workflow-run-execution-performance.yaml@main
+    uses: ./.github/workflows/workflow-run-execution-performance.yaml #aptos-labs/aptos-core/.github/workflows/workflow-run-execution-performance.yaml@main
     secrets: inherit
     with:
       GIT_SHA: ${{ github.event.pull_request.head.sha || github.sha }}
       RUNNER_NAME: executor-benchmark-runner
 
   spot-runner-execution-performance:
-    uses: aptos-labs/aptos-core/.github/workflows/workflow-run-execution-performance.yaml@main
+    uses: ./.github/workflows/workflow-run-execution-performance.yaml  #aptos-labs/aptos-core/.github/workflows/workflow-run-execution-performance.yaml@main
     secrets: inherit
     with:
       GIT_SHA: ${{ github.event.pull_request.head.sha || github.sha }}
-      RUNNER_NAME: spot-runner
\ No newline at end of file
+      RUNNER_NAME: spot-runner
```

Comment:
need to rebase I believe.

Reply:
Aah, yes, thanks @ibalajiarun 😄 Let me rebase!